common/app/slices/FixedContainers.scala

```diff
@@ -82,7 +82,8 @@ object FixedContainers {
     ("fixed/small/slow-IV", fixedSmallSlowIV),
     ("fixed/small/slow-V-half", slices(Hl4Half)),
     ("fixed/small/slow-V-third", slices(QuarterQuarterHl3)),
-    ("fixed/small/slow-V-mpu", slices(TTlMpu)),
+    ("fixed/small/slow-V-mpu", slices(Seq(TTlMpu),
+      slicesWithoutMpu = Seq(QuarterQuarterQuarterQuarter))),
```

Comment:
This is the first place where a container is defined with default slices and slices without MPU.